src/components/errorBoundary.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+import React, { Component } from 'react'
+
+export class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { hasError: false }
+  }
+
+  static getDerivedStateFromError(error) {
+    // Update state so the next render will show the fallback UI.
+    return { hasError: true }
+  }
+
+  componentDidCatch(error, errorInfo) {
+    // You can also log the error to an error reporting service
+    logErrorToMyService(error, errorInfo)
```

Comment:
logErrorToMyServer is not defined. It will fail/throw an exception after the actual exception was thorwn